interfaces/console/src/app/console/home/page.tsx

```diff
@@ -163,22 +150,31 @@ export default function Page() {
             <LoadingWrapper
               radius="small"
               width={'100%'}
-              isLoading={networkNodesLoading}
+              isLoading={networkNodesLoading || sitesLoading || nodesLoading}
             >
               <NetworkMap
                 id="network-map"
                 zoom={10}
                 className="network-map"
-                markersData={{ nodes: [], networkId: '' }}
+                markersData={{
+                  nodes:
+                    sitesRes && sitesRes?.getSites.sites.length > 0
+                      ? nodesData?.getNodesLocation.nodes || []
+                      : [],
+                }}
               >
                 {() => (
                   <>
-                    <LabelOverlayUI name={network.name} />
                     <SitesTree
-                      sites={[]}
-                      // sites={structureNodeSiteDate(
-                      //   networkNodes?.getNodesByNetwork.nodes ?? [],
-                      // )}
+                      sites={structureNodeSiteDate(
+                        {
+                          nodes:
+                            sitesRes && sitesRes?.getSites.sites.length > 0
+                              ? nodesData?.getNodesLocation.nodes || []
+                              : [],
+                        },
+                        { sites: sitesRes?.getSites.sites || [] },
+                      )}
                     />
                     {/* <SitesSelection
```

Comment:
Why is this code commented out? If it's not being used, I think we should remove it.

Reply:
No, It's gonna be need to complete the feature but right now we might not able to finish it